web/src/settings_profile_fields.js

```diff
@@ -72,7 +72,7 @@ function delete_profile_field(e) {
     e.preventDefault();
     e.stopPropagation();
 
-    const profile_field_id = Number.parseInt($(e.currentTarget).attr("data-profile-field-id"), 10);
+    const profile_field_id = Number.parseInt($(this).attr("data-profile-field-id"), 10);
```

Comment:
So, `this` is a good thing to use inside a localized click handler, but I think it's generally a better pattern to pass a more explicit parameter, but I guess you switch to that pattern in the TS migration commit, so seems fine.